cli/parser/parser.go

```diff
@@ -580,16 +580,28 @@ func ResolveArgs(parsedArgs *parsed.OrderedArgs) (*parsed.OrderedArgs, error) {
 	return resolveArgs(parsedArgs, ws)
 }
 
-// resolveArgs removes all rc-file options from the args, appends an
-// `ignore_all_rc_files` option to the startup options, parses those rc-files
-// into Configs using the default parser, and expands all config options (as
-// well as any `enable_platform_specific_config` option, if one exists) using
-// those coonfigs, and returns the result.
 func resolveArgs(parsedArgs *parsed.OrderedArgs, ws string) (*parsed.OrderedArgs, error) {
 	p, err := GetParser()
 	if err != nil {
 		return nil, err
 	}
+	return p.resolveArgs(parsedArgs, ws)
+}
+
+// ResolveArgs removes all rc-file options from the args, appends an
+// `ignore_all_rc_files` option to the startup options, parses those rc-files
+// into Configs using the default parser, and expands all config options (as
```

Comment:
I know the comment here was just moved from above - but any idea if "using the default parser" makes sense here? idk if we have a "non-default" parser.